src/transformers/pipelines/text_generation.py

```diff
@@ -95,6 +102,13 @@ class TextGenerationPipeline(Pipeline):
     begging for his blessing. <eod> </s> <eos>
     """
 
+    # Make sure the docstring is updated when the default generation config is changed
+    _default_generation_config = GenerationConfig(
+        max_new_tokens=256,
```

Comment:
```suggestion
        max_new_tokens=1024,
```

also think here a sensible default would be 1K, for typical summarise `X` usecases, wdyt?

Reply:
I think 1024 can be really, really long to generate - for example some Qwen models have 2k set and even on an M4 laptop I can end up waiting minutes for an output

I think having a very large default for systems that are streaming is fine, but for systems which are blocking, I'd actually advise something lower (256 already seems high to me, but I can live with it)